content/en/logs/log_collection/csharp.md

```diff
@@ -460,9 +463,12 @@ If you are using the `Microsoft.Extensions.Logging` integration, you can filter
 
 ## Agentless logging with Serilog sink
 
+<div class="alert alert-info">Since `0.2.0`, you can configure the Datadog sink by using an `appsettings.json` file with the <a href="https://github.com/serilog/serilog-settings-configuration">`Serilog.Setting.Configuration`</a> package.
```

Comment:
```suggestion
<div class="alert alert-info">Since <code>0.2.0</code>, you can configure the Datadog sink by using an <code>appsettings.json</code> file with the <a href="https://github.com/serilog/serilog-settings-configuration"><code>Serilog.Setting.Configuration</code></a> package.
```
If you want actual code formatting, you can use this approach -- there are other instances in this file / this PR, so you would want to make it consistent if so.